GenderPayGap.WebUI/Views/Viewing/EmployerDetails/Employer.cshtml

```diff
@@ -62,235 +62,274 @@
-                        <div class="body" style="margin-top: 2em">
-                            @if (organisation.GetSubmittedReports().Any() && organisation.Status == OrganisationStatuses.Retired)
-                            {
-                                <div class="govuk-warning-text">
-                                    <span class="govuk-warning-text__icon" aria-hidden="true" style="top: calc(50% - 15px)">!</span>
-                                    <strong class="govuk-warning-text__text">
-                                        <span class="govuk-warning-text__assistive">Notice</span>
-                                        This page is no longer updated. The organisation may no longer be in operation or operating under a different name
-                                    </strong>
-                                </div>
-                            }
+        <div class="column-two-thirds">
+        <div class="grid-row">
+            <div class="column-full">
+                <hr style="margin-top: 0"/>
+                <div class="body" style="margin-top: 2em">
+                    @if (organisation.GetSubmittedReports().Any() && organisation.Status == OrganisationStatuses.Retired)
+                    {
+                        <div class="govuk-warning-text">
+                            <span class="govuk-warning-text__icon" aria-hidden="true" style="top: calc(50% - 15px)">!</span>
+                            <strong class="govuk-warning-text__text">
+                                <span class="govuk-warning-text__assistive">Notice</span>
+                                This page is no longer updated. The organisation may no longer be in operation or operating under a different name
+                            </strong>
                         </div>
+                    }
+                </div>
 
-                        @await Html.PartialModelAsync(new Details
-                        {
-                            Id = "WhoNeedsToReport",
-                            LinkText = "Who is required to report, and the deadlines",
-                            SummaryPartial = "EmployerDetails/WhoNeedsToReport"
-                        })
-                        <p>
-                            For example, if the snapshot date is 5 April 2017 but the deadline is 4 April 2018 then the reporting period is 2017 - 2018.
-                        </p>
+                @await Html.PartialModelAsync(new Details
+                {
+                    Id = "WhoNeedsToReport",
+                    LinkText = "Who is required to report, and the deadlines",
+                    SummaryPartial = "EmployerDetails/WhoNeedsToReport"
+                })
+                <p>
+                    For example, if the snapshot date is 5 April 2017 but the deadline is 4 April 2018 then the reporting period is 2017 - 2018.
+                </p>
 
-                    </div>
-                </div>

```

Comment:
It looks like a whole bunch of stuff in this file got unindented by two tabs for no reason I can see and that's making it really hard to tell what's actually changed from the diff. Could you fix the indentation please?